sbin/shipper_utils/update_and_install_workers.py

```diff
@@ -57,6 +57,20 @@ def update_docker_images(user, host, worker, autograding_workers, autograding_co
         else:
             print(res.stdout)
         client = docker.from_env()
+        try:
+            # List all images
+            image_current_list = client.images.list()
+            container_list = images_str.split(', ')
+            container_list = set(container_list)
+            image_set = {image.attrs['RepoTags'][0] for image in image_current_list}
+            image_to_remove = set.difference(image_set, container_list)
+
+            # Remove images
+            for imagesRemoved in image_to_remove:
+                client.images.remove(imagesRemoved)
```

Comment:
Yeah I think this makes sense just cleaning up whats not listed on the file the next time we do an update, as long as the user doesn't see them listed